apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html

```diff
@@ -0,0 +1,87 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+>
+    <tui-tabs tuiFade>
```

Comment:
let's use `activeItemIndex` https://taiga-ui.dev/next/navigation/tabs#basic
`tuiFade` is redundant